lib/mix/lib/mix/dep/converger.ex

```diff
@@ -59,7 +59,7 @@ defmodule Mix.Dep.Converger do
     apps = Enum.map(main, & &1.app)
 
     lock_given? = !!lock
-    env = opts[:env]
+    env_target = {opts[:env], opts[:target]}
```

Comment:
By looking at this code, I'd go with `env_and_target`. It's slightly longer but `env_target` makes me thing of the target for the environment, plus the functions are called stuff like `split_by_env_and_target` so I think it would make sense.